app/src/main/res/layout/card_view.xml

```diff
@@ -0,0 +1,177 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.cardview.widget.CardView xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:card_view="http://schemas.android.com/apk/res-auto"
+    android:layout_width="170dp"
+    android:layout_height="296dp"
+    card_view:cardCornerRadius="8dp"
+    card_view:cardElevation="4dp">
+
+    <androidx.constraintlayout.widget.ConstraintLayout
+        android:id="@+id/card_view"
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        android:background="@drawable/card_gradient">
+
+        <androidx.constraintlayout.widget.Guideline
+            android:id="@+id/guideline_start"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:orientation="vertical"
+            app:layout_constraintGuide_begin="8dp" />
+
+        <androidx.constraintlayout.widget.Guideline
+            android:id="@+id/guideline_end"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:orientation="vertical"
+            app:layout_constraintGuide_end="8dp" />
+
+        <ImageView
+            android:id="@+id/movie_background_image_view"
+            android:layout_width="0dp"
+            android:layout_height="wrap_content"
+            android:onClick="onClickMoveToDetails"
+            android:src="@drawable/movie"
+            app:layout_constraintTop_toTopOf="@id/card_view"
+            card_view:layout_constraintEnd_toEndOf="@id/card_view"
```

Comment:
instead of
card_view:layout_constraintEnd_toEndOf="@id/card_view"
             card_view:layout_constraintStart_toStartOf="@id/card_view"
             
             
             use match parent